src/test/java/org/jenkinsci/plugins/workflow/support/steps/ExecutorStepTest.java

```diff
@@ -1197,6 +1197,17 @@ String n(Run<?, ?> b, String label) {
         });
     }
 
+    @Test
+    @Issue("JENKINS-60634")
+    public void tempDirVariable() throws Exception {
+        story.then(r -> {
+            WorkflowJob p = r.jenkins.createProject(WorkflowJob.class, "p");
+            p.setDefinition(new CpsFlowDefinition("node {if (isUnix()) {sh 'set -u && touch \"$WORKSPACE_TMP/x\"'} else {bat(/echo ok > \"%WORKSPACE_TMP%\\x\"/)}}", true));
```

Comment:
Note that unlike in the core (freestyle) test, there is no need to `mkdir $WORKSPACE_TMP` here: the control directory for `durable-task` is inside `p@tmp` already.